HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -2329,13 +2519,24 @@ def _test_default_storage_water_heater_values(hpxml, *expected_wh_values)
     storage_water_heaters = hpxml.water_heating_systems.select { |w| w.water_heater_type == HPXML::WaterHeaterTypeStorage }
     assert_equal(expected_wh_values.size, storage_water_heaters.size)
     storage_water_heaters.each_with_index do |wh_system, idx|
-      is_shared, heating_capacity, tank_volume, recovery_efficiency, location = expected_wh_values[idx]
+      is_shared, heating_capacity, tank_volume, recovery_efficiency, location, temperature, energy_factor, first_hour_rating = expected_wh_values[idx]
 
       assert_equal(is_shared, wh_system.is_shared_system)
       assert_in_epsilon(heating_capacity, wh_system.heating_capacity, 0.01)
       assert_equal(tank_volume, wh_system.tank_volume)
       assert_in_epsilon(recovery_efficiency, wh_system.recovery_efficiency, 0.01)
       assert_equal(location, wh_system.location)
+      assert_equal(temperature, wh_system.temperature)
+      if energy_factor.nil?
+        assert_nil(wh_system.energy_factor)
+      else
+        assert_equal(energy_factor, wh_system.energy_factor)
+      end
+      if first_hour_rating.nil?
+        assert_nil(wh_system.first_hour_rating)
+      else
+        assert_equal(first_hour_rating, wh_system.first_hour_rating)
+      end
```

Comment:
I don't think this is correct? FHR doesn't seem to ever be provided as an expected value to the `_test_default_storage_water_heater_values` method -- nor do I think it should be since it's never defaulted (and these tests are specific to checking defaulted values)?